packages/components/pie-cookie-banner/src/defs.ts

```diff
@@ -62,6 +62,8 @@ export interface CookieBannerProps {
      * Allows consumers to pass in a specific cookie technology link.
      */
     cookieTechnologiesLink: string;
+
+    defaultPreferences: Record<PreferenceIds, boolean> | object
```

Comment:
can any of these flags be omitted? for example could I pass `{ functional: true }` ? If so, I believe the booleans should be boolean | undefined (optional)